packages/block-editor/src/components/block-ratings/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Stars from './stars';
+
+export const BlockRatings = ( { rating, ratingCount } ) => (
+	<div className="block-ratings">
```

Comment:
For classNames we have some guidelines See https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#naming

This className for instance should be `block-editor-block-ratings`